src/sidebar/services/groups.js

```diff
@@ -66,7 +116,11 @@ function groups($rootScope, store, api, isSidebar, localStorage, serviceUrl, ses
         params.document_uri = uri;
       }
       documentUri = uri;
-      return api.groups.list(params);
+      return api.groups.list(params, null, { includeMetadata: true });
+    }).then(({ data, token }) => {
```

Comment:
It's functionally fine, but this promise chain is starting to feel long and harder to read. Not _terrible_, though!

Reply:
async/await is the answer to that IMO. There used to be a big code-size hit when transpiling that for old browsers using eg. regenerator. We could perhaps build separate bundles for new/old browsers or perhaps the transpilation situation has gotten better since I last looked at this.